services/licence-finder/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: '3.7'
+
+x-licence-finder: &licence-finder
+  build:
+    context: .
+    dockerfile: Dockerfile.govuk-base
+  image: licence-finder
+  volumes:
+    - ${GOVUK_ROOT_DIR:-~/govuk}:/govuk:delegated
+    - home:/home/build
+  working_dir: /govuk/licence-finder
+
+services:
+  licence-finder-lite: &licence-finder-lite
+    <<: *licence-finder
+    privileged: true
+    depends_on:
+      - mongo
+      - elasticsearch6
+    environment:
+      MONGODB_URI: "mongodb://mongo/licence-finder_development"
+      VIRTUAL_HOST: licence-finder.dev.gov.uk
+      ELASTICSEARCH_URI: http://elasticsearch6:9200
+      HOST: 0.0.0.0
+    ports:
+      - "3000"
+
+  licence-finder-app:
+    <<: *licence-finder
+    depends_on:
+      - mongo
+      - redis
+      - static-app
+      - elasticsearch6
+      - content-store-app
+    environment:
+      GOVUK_ASSET_ROOT: frontend.dev.gov.uk
```

Comment:
I'm not sure if this is right, especially as this doesn't depend on 'frontend'. Does it work without?

Reply:
This was un-needed. I'm not sure why I added it...